src/cli/credrank.js

```diff
@@ -73,4 +97,38 @@ function printCredSummaryTable(credGraph: CredGraph) {
   sortedParticipants.slice(0, 20).forEach((n) => console.log(row(n)));
 }
 
+function printCredDiffTable(credGraph: CredGraph, priorCredGraph: CredGraph) {
+  console.log(`# Top Participants By New Cred`);
+  const priorParticipants: Map<Uuid, Participant> = new Map();
+  for (const participant of priorCredGraph.participants())
+    priorParticipants.set(participant.id, participant);
+
+  const credParticipants = Array.from(credGraph.participants());
+  const sortedParticipants = sortBy(credParticipants, (p) => -p.cred);
+  function row({cred, description, id}) {
+    const prior = priorParticipants.get(id);
+    if (!prior)
+      throw `Participant [${description}, ${id}] exists in the new scores but not in the old. Rerun without -d to refresh.`;
+    const percentageChange = (100 * (cred - prior.cred)) / prior.cred;
+    const percentageChangeStr =
+      percentageChange > 10000
+        ? ">10,000%"
+        : `${percentageChange > 0 ? "+" : ""}${percentageChange.toFixed(1)}%`;
+    return {
+      "Name": description,
+      "Prior Cred": prior.cred.toFixed(1),
+      "New Cred": cred.toFixed(1),
+      "% Change": percentageChangeStr,
+    };
+  }
+  type Row = {
+    "Name": string,
+    "Prior Cred": string,
+    "New Cred": string,
+    "% Change": string,
+  };
+  const rows: Array<Row> = sortedParticipants.slice(0, 20).map((n) => row(n));
```

Comment:
nit: this type seems a little unnecessary since it's basically just a re-declaration of the return type which is just one line above, and never leaves the scope of this function. no change requested though. also technically $ReadOnlyArray would be more accurate than Array.

Reply:
flow was being weird about console.table's expectations without typing `rows` like this. Playing with it now, turns out I can put the slice/map inline in the console.table without complaint, which is much nicer.